File: Zxing/app/src/main/java/com/k_shimoju/zxing/MainActivity.java
package com.k_shimoju.zxing;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

import com.google.zxing.integration.android.IntentIntegrator;
import com.google.zxing.integration.android.IntentResult;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.btn_barcode)
    protected void btnReadIsbn() {

        IntentIntegrator integrator = new IntentIntegrator(this);
        integrator.initiateScan();
    }

    @Override
    public void onActivityResult(int requestCode, int resultCode, Intent data) {

        IntentResult intentResult = IntentIntegrator.parseActivityResult(requestCode, resultCode, data);
        String code;

        if (null != intentResult && (code = intentResult.getContents()) != null) {
            Toast.makeText(this, code, Toast.LENGTH_SHORT).show();
        }
    }
}

File: All/app/src/main/java/com/all/app/android/all/ui/MainActivity.kt
package com.all.app.android.all.ui

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import androidx.navigation.Navigation
import androidx.navigation.ui.NavigationUI
import androidx.navigation.ui.setupWithNavController
import com.all.app.android.all.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {//, HasSupportFragmentInjector {

    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        navigation.setupWithNavController(Navigation.findNavController(this, R.id.main_contents))
        navigation.setOnNavigationItemSelectedListener { item ->
            NavigationUI.onNavDestinationSelected(item, Navigation.findNavController(this, R.id.main_contents))
        }
    }
}
